octoprint_mrbeam/templates/settings/serialconnection_settings.jinja2

```diff
@@ -28,7 +28,8 @@
                 <div class="controls">
                     <div class="input-append">
                         <input type="number" step="any" min="0" class="input-mini text-right" data-bind="value: serial_timeoutCommunication" id="settings-serialTimeoutCommunication">
-                        <span class="add-on">s</span>
+                        {# Translators: seconds #}
```

Comment:
this file should not be translated, it is not used at all.
We should consider to remove it from the code.

Reply:
Removed all string markings from the file